.gitlab-ci.yml

```diff
@@ -10,14 +10,19 @@ variables:
   rules:
     # Automatically run the pipeline for all pushed tags
     - if: $CI_COMMIT_TAG
+  id_tokens:
+    DDSIGN_ID_TOKEN:
+      aud: image-integrity
```

Comment:
Oh nice, TIL about this new feature! https://docs.gitlab.com/ee/ci/secrets/id_token_authentication.html